Mollie/API/Client.py

```diff
@@ -41,9 +41,12 @@ def __init__(self, api_key=None, api_endpoint=None):
         self.api_version = self.API_VERSION
         self.api_key = self.validateApiKey(api_key) if api_key else None
         self.payments = Resource.Payments(self)
-        self.payment_refunds = Resource.Refunds(self)
-        self.issuers = Resource.Issuers(self)
+        self.payment_refunds = Resource.PaymentRefunds(self)
+        self.payment_chargebacks = Resource.PaymentChargebacks(self)
         self.methods = Resource.Methods(self)
+        self.issuers = Resource.Issuers(self)
+        self.refunds = Resource.Refunds(self)
+        self.chargebacks = Resource.Chargebacks(self)
         self.customers = Resource.Customers(self)
         self.customer_mandates = Resource.Mandates(self)
```

Comment:
`customer_mandates` and `customer_subscriptions` are now the only inconsistently named classes. Would it be possible to update these as well to `CustomerMandates` and `CustomerSubscriptions`?

This is of course not blocking for the merge.